projects/sandbox/train/train/train.py

```diff
@@ -204,9 +234,56 @@ def main(
     # into one file for simplicity
     background = load_background(hanford_background, livingston_background)
     if valid_frac is not None:
-        background, valid_background = split(background, 1 - valid_frac, 1)
-        valid_injector.fit(H1=background[0], L1=background[1])
-        valid_waveforms, _ = valid_injector.sample(-1)
+        # split up our background data into train and validation splits
+        background, valid_background = split(background, 1 - valid_frac, -1)
```

Comment:
Yeah in this case the background array is 2D so those should be equivalent. I don't know exactly why I opted for this change, but now that I'm thinking about it I do like the idea of `-1` as a handy shorthand in our context for "do this operation along the time dimension," since that's where it will be regardless of the shape of the array